Source/SGDLib/SimpleDistGradAggregator.h

```diff
@@ -326,36 +305,100 @@ class SimpleDistGradAggregator : public IDistGradAggregator<ElemType>
         if (!m_mpi->IsMainNode())
             m_mpi->Isend(headerCPU, headerCPU->Size(), MPI_CHAR, m_mpi->MainNodeRank(), numGradMatrices, &sendHeaderRequest) || MpiFail("MPI_Isend");
 
+
+        // New aggregation pipeline for non-GDR
         // Perform async allreduce on the gradient data
```

Comment:
Yeah. My comment is not accurate. For non-GDR & GPU, we use sync allreduce; but for CPU, I make no modification, still async allreduce. I will update the words.